api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst

```diff
@@ -0,0 +1,170 @@
+.. _put-update-identity-provider-v2.0:
+
+Update IDP
+~~~~~~~~~~
+
+.. code::
+
+   PUT /v2.0/RAX-AUTH/federation/identity-providers/{identityProviderId}
+
+Update an Identity provider.
+
+.. note::
+
+   - User-admin or User-manage can make a request only when caller's domain is the same as the specified Identity Provider's (IDP's) approvedDomainId.
+   - User-admin or User-manage can update the name, description and emailDomains. Any specified values for other fields are ignored.
```

Comment:
User-admin or User-manage roles can update the name, description,and emailDomains.